kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/security/auth/SimpleAclAuthorizer.java

```diff
@@ -127,7 +111,64 @@ protected PulsarService getPulsarService() {
                     });
 
         });
+        return permissionFuture;
+    }
+
+    private void authoriseTopicOverNamespacePolicies(KafkaPrincipal principal, AuthAction action,
+                                                     CompletableFuture<Boolean> permissionFuture,
+                                                     TopicName topicName, Optional<Policies> policies) {
```

Comment:
The `Optional<Policies> policies` can be replace with `Policies policies` now.

Reply:
right !
Fixed and pushed